components/raftstore-v2/src/operation/command/write/mod.rs

```diff
@@ -269,54 +270,191 @@ impl<EK: KvEngine, R: ApplyResReporter> Apply<EK, R> {
 
         let start_key = keys::data_key(start_key);
 
-        info!(
-            self.logger,
-            "execute delete range";
-            "range_start" => log_wrappers::Value::key(&start_key),
-            "range_end" => log_wrappers::Value::key(&end_key),
-            "notify_only" => notify_only,
-            "use_delete_range" => use_delete_range,
-        );
-
+        let start = Instant::now_coarse();
         // Use delete_files_in_range to drop as many sst files as possible, this
         // is a way to reclaim disk space quickly after drop a table/index.
         if !notify_only {
```

Comment:
Just be curious: when will notify_only be true and what's the purpose of this fn when notify_only is true. It seems does not really do any delete.

Reply:
See https://github.com/tikv/tikv/pull/4490